source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -102,9 +102,10 @@ the feature cannot determine what to do and the operation will fail.
 
 * The cluster's management network stays on the original interface which is excluded from the operation. 
 
-* QoS, DNS and custom_properties are currently unsupported.
+* Copying QoS, DNS and custom_properties.
```

Comment:
@bellle could you please add "are not supported" to the end of the line?
Just to make it super clear.

Reply:
Done